api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/saml/idp/SamlIdPLogoutProperties.java

```diff
@@ -57,4 +58,9 @@ public class SamlIdPLogoutProperties implements Serializable {
      * Whether SAML SLO is enabled and processed.
      */
     private boolean singleLogoutCallbacksDisabled;
+
+    /**
+     * The order in which the logout request binginds should be tried (if available at the SP level).
+     */
+    private List<String> logoutRequestBindings;
```

Comment:
I think should consider adding the defaults here. This has the advantage that documentation will also automatically list the values. 

You don't need to refer to the bindings by their consts that would force you to bring in the opensaml library; using the string value should be OK. (They never ever change)

Reply:
Sure. This is a better option (default string values). I just updated the PR.